lazy.config.ts

```diff
@@ -43,82 +43,86 @@ export function generateSharedTasks(bublic: '<rootDir>' | '<rootDir>/bublic') {
 				inputs: ['package.json', `${bublic}/scripts/refresh-assets.ts`, `${bublic}/assets/**/*`],
 			},
 		},
-		'build:types': {
+		'build-types': {
 			execution: 'top-level',
 			baseCommand: `tsx ${bublic}/scripts/typecheck.ts`,
 			cache: {
 				inputs: {
-					include: [
-						'{,bublic/}packages/*/src/**/*.{ts,tsx}',
-						'{,bublic/}{apps,scripts,e2e}/**/*.{ts,tsx}',
-						'{,bublic/}{apps,packages}/*/tsconfig.json',
-						'{,bublic/}{scripts,e2e}/tsconfig.json',
-					],
-					exclude: ['**/dist*/**/*.d.ts'],
+					include: ['<allWorkspaceDirs>/**/*.{ts,tsx}', '<allWorkspaceDirs>/tsconfig.json'],
+					exclude: ['<allWorkspaceDirs>/dist*/**/*', '<allWorkspaceDirs>/api/**/*'],
 				},
+				outputs: ['<allWorkspaceDirs>/*.tsbuildinfo', '<allWorkspaceDirs>/.tsbuild/**/*'],
 			},
 			runsAfter: {
 				'refresh-assets': {},
 				'maybe-clean-tsbuildinfo': {},
 			},
 		},
-		'build:api': {
+		'build-api': {
 			execution: 'independent',
 			cache: {
 				inputs: ['.tsbuild/**/*.d.ts', 'tsconfig.json'],
+				outputs: ['api/**/*'],
```

Comment:
@SomeHats I added initial support for output caching to lazyrepo over the weekend, testing it out gingerly on our api build step.

Reply:
(still only local)